Formula/openjdk.rb

```diff
@@ -5,6 +5,11 @@ class Openjdk < Formula
   sha256 "9c5be662f5b166b5c82c27de29b71f867cff3ff4570f4c8fa646490c4529135a"
   license :cannot_represent
 
+  livecheck do
+    url "https://hg.openjdk.java.net/jdk-updates/jdk#{version.major}u/tags"
+    regex(/jdk-(\d+\.?\d*\.?\d*)-ga/i)
```

Comment:
```suggestion
    regex(/>\s*?jdk[._-]v?(\d+(?:\.\d+)*)-ga\s*?</i)
```

This brings the regex more in line with existing standards and anchors the beginning/end, so we only match the text in links (not any text on the page).

Reply:
Done